packages/react-native/src/types/Module.ts

```diff
@@ -454,13 +449,10 @@ export interface Module {
 
   /**
    * Get the current notification settings for this application on the current device.
-   *
-   * On Android, all of the properties on the `IOSNotificationSettings` interface response return
-   * as `AUTHORIZED`.
-   *
-   * @platform ios
+   * On Android, `authorizationStatus` will return only either `IOSAuthorizationStatus.DENIED` or `IOSAuthorizationStatus.AUTHORIZED`
```

Comment:
same issue here - seems strange that in a new world where we have Android status as well, we are tied to an IOS-specific object

Reply:
yes, I believe this should be renamed to `AuthorizationStatus`

Reply:
@helenaford should we introduce name updates as part of another PR?